tests/testExamples.m

```diff
@@ -11,66 +12,88 @@
             'DSPCMonolayerStandardLayers',...
             'DSPCMonolayerTwoContrasts',...
             'orsoDSPCCustomLayers',...
-            'projectConversion'};
+            'projectConversion',...
+            'DSPCMonolayerCustomLayers',...
+            'controlAlgorithms'};
```

Comment:
The "controlAlgorithms" test adds considerably to the runtime and processing requirements for the test suite. Does it need to be run in full every time?